### ISSUE TITLE
Sobol experimental design

### ISSUE THREAD

**Issue by AnthonyLarroque** [amzn/emukit#278]
Hello,

Since I think the sobol experimental design was not implemented in emukit but was in GPyOpt, I developed it for emukit following the LatinDesign model. Even if it is not a crucial contribution, I am sending it to you in .txt format if you would like to include it in the official version of emukit:

[sobol_design.txt](https://github.com/amzn/emukit/files/4064785/sobol_design.txt)

Kind regards,
Anthony Larroque

**Comment by apaleyes** [comment]
Thanks! Any reason you didn't cut a PR for this?

**Comment by AnthonyLarroque** [comment]
Hi @apaleyes,

First of all, because I am not fully comfortable with GitHub and the issue was the easiest way for me to do that. I know Git is useful but until now, I did not work a lot with that. I will have to learn it properly at some point. 

Secondly, because I also made some changes in the examples.gp_bayesian_optimization module to include other features in GPBayesianOptimization and if I am not wrong, these differences would also be included in the pull request. For clarity reasons, I just decided to send the sobol_design.

Regards,
Anthony Larroque
